### PR TITLE
Move under the sprint-guide subdirectory

### PR DESCRIPTION
**Pull request by cpytel**
We will ultimately hosting this at design.thoughtbot.com/sprint-guide.
This moves under that subdirectory, which also opens up a place for the
rest of design.thoughtbot.com to live.

We will eventually merge the repositories, either design-dot will come
in here or the other way around.

I've opten to keep data and styles up at the root so they can be shared
between everything, but I don't feel strongly about that and I'm open to
feedback.

This will be a pretty disruptive change to any in-progress work, so I'm
happy to wait to merge until there is a good time to do that, though I'd
prefer not to let this branch live for too long as then rebasing will
become difficult.

### CODE REVIEW

**Review by vendelal — APPROVED** [review]
I like the way you've kept docs and styles up at the root and agree with your reasoning. I'm not sure how disruptive the changes will be in terms of rebasing, but https://github.com/thoughtbot/design-sprint-guide/pull/59, https://github.com/thoughtbot/design-sprint-guide/pull/74, and https://github.com/thoughtbot/design-sprint-guide/pull/67 touch the most amount of files. It could be worth waiting until those are merged to make this change.

**Comment by davidrnovak** [comment]
@cpytel I merged the other big PR we were waiting for so I think we're in a pretty good spot for the great rebasing to begin.

**Pull request merged**
